zeppelin-web/src/components/navbar/navbar.controller.js

```diff
@@ -86,12 +86,19 @@ function NavCtrl ($scope, $rootScope, $http, $routeParams, $location,
     websocketMsgSrv.getHomeNote()
   }
 
-  function logout () {
+  function logout() {
     let logoutURL = baseUrlSrv.getRestApiBase() + '/login/logout'
 
     // for firefox and safari
     logoutURL = logoutURL.replace('//', '//false:false@')
-    $http.post(logoutURL).error(function () {
+
+    $http.post(logoutURL).then(function () {}, function (response) {
+      if (response.data) {
+        let res = angular.fromJson(response.data).body
+        if (res['redirectURL']) {
+          window.location.href = res['redirectURL'] + window.location.href
+        }
+      }
       // force authcBasic (if configured) to logout
       $http.post(logoutURL).error(function () {
```

Comment:
this used to be part of basic auth I guess? shall we check if that is still working?

Reply:
Have checked that again, and authentication with `authcBasic` still works fine.